src/TorchSharp/TorchSharp.csproj

```diff
@@ -23,6 +23,8 @@
   <ItemGroup>
     <PackageReference Include="Google.Protobuf" Version="3.21.5" />
     <PackageReference Include="SharpZipLib" Version="1.3.3" />
+    <PackageReference Include="SkiaSharp" Version="2.88.3" />
+    <PackageReference Include="SkiaSharp.NativeAssets.Linux.NoDependencies" Version="2.88.3" />
```

Comment:
IMHO creating a dependency from TorchSharp to Skia is a bad idea. There are pure .NET image libraries that could be used instead, including legacy `System.Drawing`, `ImageSharp`, and many more. Using Skia adds native library dependencies further restricting platforms where TorchSharp can run.

Reply:
SkiaSharp has good platform coverage. It's only meant to be the default image reader/writer, and it's only being used for encode/decode, not for any processing. Anyone can extend with their preferred alternative, and there's already an example using ImageSharp.

Reply:
@NiklasGustafsson Adding a dependency to Skia makes it more difficult to use. However, if we have separated packages like `TorchSharp.Vision` and `TorchSharp.Audio`, then I think adding a dependency to Skia it not bad. At the end, `TorchSharp.Vision` and `TorchSharp.Audio` have to depend on more I/O and utility libraries.

Reply:
Hmm... maybe the time has come to separate out TorchVision and TorchAudio in to their own packages. There's not much in TorchText at this point...

Reply:
- `system.drawing` will be phased out in .NET7 due to windows OS specific and ncompatibility with other OS 
- `imagesharp` is a serious option to substitute `system.drawing`. Unless Microsoft plans to acquire the company behind imagesharp, skiasharp seems to be a reasonable choice for now.

Reply:
> maybe the time has come to separate out TorchVision and TorchAudio into their own packages.

I agree that it is time for TorchSharp to be compatible with PyTorch in terms of packages and file structure arrangements to ease the process of **identifying yet-to-implement gaps** and of porting more PyTorch codes to TorchSharp.

Reply:
> There's not much in TorchText at this point...

We need to continue this discussion.  

Tentatively, I see no reason why we need to adhere to TorchText but perhaps we could go around it and have a synergistic strategy of addressing unmet needs for deep NLP for .NET communities. This means a coordinated strategy is needed from TorchSharp, ML.NET, Onnx runtime, and extension.

Reply:
Okay, so what I'll do it merge this, and then start a PR to separate TV and TA from the core TorchSharp package. The next release won't go out until the second PR comes in.

Reply:
Even if separated, what is the advantage of having Skia vs ImageSharp?